updatecli/weekly.d/contributing.yaml

```diff
@@ -0,0 +1,82 @@
+    spec:
+      file: hieradata/common.yaml
+      key: $.profile::jenkinscontroller::jcasc.agent_images.azure_vms_gallery_image.version
+
+  getToolVersions:
+    kind: yaml
+    name: Get tool versions from packer-images
+    dependson:
+      - getPackerImageVersion
+    spec:
+      file: https://raw.githubusercontent.com/jenkins-infra/packer-images/{{ source "getPackerImageVersion" }}/provisioning/tools-versions.yml
+
+conditions:
+  checkMavenAvailability:
+    kind: shell
+    disablesourceinput: true
+    spec:
+      command: curl --connect-timeout 5 --location --head --fail --silent --show-error https://archive.apache.org/dist/maven/maven-3/{{ source "getToolVersions" "maven.version" }}/binaries/apache-maven-{{ source "getToolVersions" "maven.version" }}-bin.tar.gz
+
+  checkJDKAvailability:
+    kind: shell
+    disablesourceinput: true
+    spec:
+      command: curl --connect-timeout 5 --location --head --fail --silent --show-error https://adoptium.net/temurin/releases/v{{ source "getToolVersions" "jdk.version" }}
+
+  checkPackerImageExists:
+    kind: http
+    name: Verify packer-images tools-versions.yml exists
+    spec:
+      url: https://raw.githubusercontent.com/jenkins-infra/packer-images/{{ source "getPackerImageVersion" }}/provisioning/tools-versions.yml
+
+targets:
+  updateTechnicalRequirements:
+    name: Update Technical Requirements section in CONTRIBUTING.md
+    kind: file
+    spec:
+      file: CONTRIBUTING.md
+      transformers:
+        - replaceRegex:
+            pattern: '### Technical Requirements[\s\S]*?(?=### Add a)'
+            with: |
+              ### Technical Requirements
+
+              In order to work with that repository you will need:
+
+              - JDK {{ source "getToolVersions" "jdk.version" }}: Check [Temurin releases](https://adoptium.net/temurin/releases/v{{ source "getToolVersions" "jdk.version" }})
+              - Maven {{ source "getToolVersions" "maven.version" }}: Check [Maven releases](https://maven.apache.org/download.cgi)
+              - Groovy: https://groovy-lang.org/install.html
+
+              The CI environment uses the `{{ source "getMavenLabel" }}` container image which is defined in the [jenkins-infra repository](https://github.com/jenkins-infra/jenkins-infra/blob/production/hieradata/common.yaml) and built using the [packer-images](https://github.com/jenkins-infra/packer-images) repository.
+
+              For reference, the exact versions used in our CI environment are defined in:
+              * [Packer Image Version](https://github.com/jenkins-infra/jenkins-infra/blob/production/hieradata/common.yaml): {{ source "getPackerImageVersion" }}
+              * [Tools Versions](https://raw.githubusercontent.com/jenkins-infra/packer-images/{{ source "getPackerImageVersion" }}/provisioning/tools-versions.yml)
+
+              These versions are automatically kept in sync with our CI environment through updatecli.
```

Comment:
Same a above: these URL are subject to change and it would make a lot of PRs to the doc (2 per eek) with no additional value.

=> I suggest to keep only the repository's URL (and eventually propose an update to the packer image README to mention the tools version if not already the case), and/or link to https://github.com/jenkins-infra/documentation/blob/main/ci.adoc#tools-labels.
OR remove the block and keep the existing doc (except it's kept up to date)